lldb/source/API/SBValue.cpp

```diff
@@ -1259,7 +1259,7 @@ lldb::SBValue SBValue::EvaluateExpression(const char *expr,
   return result;
 }
 
-bool SBValue::GetDescription(SBStream &description) {
+bool SBValue::GetDescription(SBStream &description, bool short_mode) {
```

Comment:
I wonder if we need to add a `SBValueDescriptionOptions` that includes a few more of the potential values for customizing the description. 

Maybe that. can be a separate PR, but I wonder if we could use this to simplify VariableDescription and maybe have it generate the auto-summaries.

Reply:
I think the `DescriptionLevel` from my previous comment should be sufficient. If we know of other things that we want to add, having an options structure could still be better.